### DISCUSSION TITLE
How do I "unselect" a normal Button after click?

### DISCUSSION THREAD

**Discussion by tomsoja978** [primefaces/community#905] [PrimeReact]
I don't know if it is related to the theme i am using (lara-dark-purple) but if i click on a normal Button ( https://primereact.org/button/ ) the outline will remain, resulting in a different appearance than before the click and leading to missunderstandings for the user.

I expected this Button to be stateless and appearing in the same way after the click before it was clicked. 

![image](https://github.com/primefaces/community/assets/144290567/7dfa04f7-599c-446a-bfbd-e5b644f35529)

**I do not want a toggle button. I want the button to look in the same way as it did before the click**

As melloware mentioned, the button is getting focus, apparently. I don't want my buttons to get focus when i click them.

**Comment by melloware** [comment]
It looks like the button has "focus" since it was the last item clicked.   You can either try and remove the focus CSS or on your `click` of button simply set the focus somewhere else.

**Reply by tomsoja978** [reply]
Can you please explain me how to do that? I think the button's css class is .p-button .p-button-outlined  (according to chrome inspection)

**Comment by mertsincan** [comment]
I think you can use ToggleButton component; https://primereact.org/togglebutton/
